kubernetes/cache.go

```diff
@@ -0,0 +1,458 @@
+package kubernetes
+
+import (
+	"errors"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/kiali/kiali/log"
```

Comment:
this import should be below the third-party imports in the "kiali import section" to match coding style